src/main/java/core/basesyntax/dao/FruitDaoCsvImpl.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.FruitTransaction;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitDaoCsvImpl implements FruitDao {
+    public static final String FILE_NAME = "src/main/resources/fruits.csv";
+    private static final String FILE_REPORT = "src/main/resources/report.csv";
+
+    @Override
+    public void writeToFile(String report) {
+
+        try (FileWriter writer = new FileWriter(FILE_REPORT);
+                BufferedWriter bwr = new BufferedWriter(writer)) {
+            bwr.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write file " + FILE_REPORT);
+        }
+    }
+
+    @Override
+    public List<FruitTransaction> readFromFileToArray() {
```

Comment:
this is a separate task  - create a service interface and its impl - something like TransactionParser. Here, we digest raw data, and transform it into our model - FruitTransaction